crypt.5

```diff
@@ -155,6 +155,7 @@ and does not show the division into prefix, options, salt, and hash.
 .br
 yescrypt is a scalable passphrase hashing scheme designed by Solar
 Designer, which is based on Colin Percival's scrypt.
+Recommended for new password hashes.
```

Comment:
I've been trying to avoid using the word "password" in the documentation (because using just a single word isn't really enough anymore).  In this file, "Recommended for new hashes" would be fine, as we had before under bcrypt's entry; or, if you think it's clearer that way, "Recommended for new passphrase hashes."